Primo 2018.js

```diff
@@ -0,0 +1,428 @@
+{
+	"translatorID": "cd669d1f-96b8-4040-aa36-48f843248399",
+	"label": "Primo 2018",
+	"creator": "Philipp Zumstein",
+	"target": "^https?://[^.]*.hosted.exlibrisgroup.com/primo-explore/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2018-01-09 17:31:37"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2018 Philipp Zumstein
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+
+// attr()/text() v2
+function attr(docOrElem,selector,attr,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.getAttribute(attr):null;}function text(docOrElem,selector,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.textContent:null;}
+
+
+function detectWeb(doc, url) {
+	var rows = doc.querySelectorAll('.urlToXmlPnx[data-url]');
+	if (rows.length>=1) return "multiple";
```

Comment:
I did this on purpose because for a single item we still have to call the PNX url. By detecting this as part of multiple, we don't need any additional code but can also use the code for multiples there.

Well I guess we can also add small things to make this work. However, then we would show a book item always, okay? Also for all articles and non-book material...

Reply:
Yes, I think that's we do with current Primo and I think it's the right approach -- just show a book icon for all singles.